packages/api-v2/package.json

```diff
@@ -29,7 +29,7 @@
     "@nestjs/passport": "^8.1.0",
     "@nestjs/platform-express": "^8.0.0",
     "@nestjs/typeorm": "^8.0.2",
-    "bcrypt": "^5.0.1",
+    "bcrypt": "5.0.0",
```

Comment:
Maybe add a comment noting why we downgrade `bcrypt`? (5.0.1 not working with M1, giving me this error `mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e')`)

Reply:
ahh yea, adding comments is weird with package.json... but we should be good if people reviewing this pr see your comment here for some context